compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/awt/ComposeWindowTest.kt

```diff
@@ -151,6 +152,12 @@ class ComposeWindowTest {
                 assertThat(window.preferredSize).isEqualTo(Dimension(234, 345))
                 window.pack()
                 assertThat(window.size).isEqualTo(Dimension(234, 345))
+
+                assertThat(window.scene.platform.windowInfo.containerSize)
+                    .isEqualTo(IntSize(
+                        width = (234 * window.density.density).toInt(),
+                        height = (345 * window.density.density).toInt(),
+                    ))
```

Comment:
I think it's better to test `WindowInfo` separately from other things. A good test should validate one thing.

Also, don't you want to test it inside the composition (i.e. with `LocalWindow.current.containerSize`?